app/scripts/modules/ecs/src/serverGroup/configure/serverGroupConfiguration.service.ts

```diff
@@ -113,6 +113,12 @@ export interface IEcsServiceDiscoveryRegistryAssociation {
   containerName: string;
 }
 
+export interface IEcsCapacityProviderStrategy {
```

Comment:
since the "strategy" is the list of providers & their respective values, shouldn't we call this [`IEcsCapacityProviderStrategyItem`](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_CapacityProviderStrategyItem.html) ?

Reply:
Resolved.